source/Account.cpp

```diff
@@ -276,7 +276,7 @@ string Account::Step(int64_t assets, int64_t salaries, int64_t maintenance)
 			out << creditString(mortgagesPaid) << " in mortgages"
 				<< (finesPaid ? " and " : ".");
 		if(finesPaid)
-			out << creditString(finesPaid) << " in fines.";
+			out << creditString(finesPaid) << " in other payments.";
```

Comment:
Rather than piggybacking on fines, I think this would be worth adding as its own entry.

It's useful for the player to be able to see that they have outstanding profit shares, and it'll become even more useful in the future when these things affect morale.

Reply:
I agree completely with this and will look into it.  This piggybacking is just how ES did this way back when DBs were a thing.